browser/resources/settings/brave_appearance_page/toolbar.html

```diff
@@ -1,25 +1,86 @@
-<style include="settings-shared iron-flex">
+<style include="cr-shared-style settings-shared md-select iron-flex">
+  /* Lines up with cr-input. */
+  #custom-input {
+    /* (cr-input line-height + cr-input top/bottom padding) / 2 -
+       cr-radio disc-wrapper height / 2 */
+    --cr-radio-button-disc-margin-block-start: calc(
```

Comment:
FWIW, I think a few of these values might get changed when upstream flips the 2023 refresh flag - `cr-input` line-height is getting changed to `16px` and some of the others might be affected too.